ui/src/components/About/__fixtures__/AboutView.fixture.tsx

```diff
@@ -0,0 +1,8 @@
+import { MemoryRouter } from "react-router";
+import AboutView from "../AboutView";
+
+export default function AboutViewFixture () {
+  return (
+      <AboutView />
```

Comment:
This one fails to render for me when set to the mobile viewport, with this error:

```
Ouch, something wrong!
useHref() may be used only in the context of a <Router> component.

    at LinkWithRef (http://localhost:5050/apps/groups/node_modules/.vite/deps/react-router-dom.js?v=0567a300:325:5)
    at div
    at div
    at MobileHeader (http://localhost:5050/apps/groups/src/components/MobileHeader.tsx:20:3)
    at header
    at div
    at Layout (http://localhost:5050/apps/groups/src/components/Layout/Layout.tsx:19:3)
    at AboutView (http://localhost:5050/apps/groups/src/components/About/AboutView.tsx:44:3)
    at AboutViewFixture
    at FixtureElement (http://localhost:5050/apps/groups/node_modules/.vite/deps/chunk-SOOXLD3I.js?v=0567a300:9646:27)
    at FixtureCapture (http://localhost:5050/apps/groups/node_modules/.vite/deps/chunk-SOOXLD3I.js?v=0567a300:9632:27)
    at div
    at div
    at QueryClientProvider (http://localhost:5050/apps/groups/node_modules/.vite/deps/chunk-P4JGML6C.js?v=0567a300:2658:3)
    at Provider (http://localhost:5050/apps/groups/node_modules/.vite/deps/chunk-DMYY67AS.js?v=0567a300:20:15)
    at $a093c7e1ec25a057$export$f78649fb9ca566b8 (http://localhost:5050/apps/groups/node_modules/.vite/deps/@radix-ui_react-tooltip.js?v=0567a300:866:11)
    at AppDecorator (http://localhost:5050/apps/groups/src/cosmos.decorator.tsx:23:3)
    at ErrorCatch (http://localhost:5050/apps/groups/node_modules/.vite/deps/react-cosmos-plugin-vite_dom.js?v=0567a300:182:5)
    at DecoratedFixture (http://localhost:5050/apps/groups/node_modules/.vite/deps/chunk-SOOXLD3I.js?v=0567a300:9658:29)
    at FixtureProvider (http://localhost:5050/apps/groups/node_modules/.vite/deps/chunk-SOOXLD3I.js?v=0567a300:9672:25)
    at FixtureModule (http://localhost:5050/apps/groups/node_modules/.vite/deps/chunk-SOOXLD3I.js?v=0567a300:9724:26)
    at StaticModuleLoader (http://localhost:5050/apps/groups/node_modules/.vite/deps/chunk-SOOXLD3I.js?v=0567a300:9773:31)
    at RendererSync (http://localhost:5050/apps/groups/node_modules/.vite/deps/chunk-SOOXLD3I.js?v=0567a300:9785:25)
    at FixtureLoaderConnect (http://localhost:5050/apps/groups/node_modules/.vite/deps/chunk-SOOXLD3I.js?v=0567a300:9842:33)
    at ClientFixtureLoader (http://localhost:5050/apps/groups/node_modules/.vite/deps/chunk-SOOXLD3I.js?v=0567a300:9864:32)
    at RendererProvider (http://localhost:5050/apps/groups/node_modules/.vite/deps/chunk-SOOXLD3I.js?v=0567a300:9878:60)
    at StatefulRendererProvider (http://localhost:5050/apps/groups/node_modules/.vite/deps/chunk-SOOXLD3I.js?v=0567a300:9906:37)
    at DomRendererProvider (http://localhost:5050/apps/groups/node_modules/.vite/deps/react-cosmos-plugin-vite_dom.js?v=0567a300:166:32)
    at DomFixtureLoader (http://localhost:5050/apps/groups/node_modules/.vite/deps/react-cosmos-plugin-vite_dom.js?v=0567a300:226:29)
Check console for more info.
```

Reply:
Fixed in fbc8924